ql/src/java/org/apache/hadoop/hive/ql/exec/GroupByOperator.java

```diff
@@ -712,6 +751,12 @@ private void processKey(Object row,
 
   @Override
   public void process(Object row, int tag) throws HiveException {
+    if (hashAggr) {
+      if (getConfiguration().get("forced.streaming.mode", "false").equals("true")) {
```

Comment:
This should go into a GroupbyDesc

Reply:
i have removed it in the next commit ..had added for test only.